pipeline/inferrer/feature_training/src/aws.py

```diff
@@ -1,28 +1,39 @@
 import boto3
+import os
+import requests
 from botocore.exceptions import ClientError
+from .logging import get_logger
+
+logger = get_logger(__name__)
+
+
+def get_ecs_container_metadata():
+    endpoint = os.environ["ECS_CONTAINER_METADATA_URI_V4"]
+    req = requests.get(endpoint)
+    return req.json()
```

Comment:
You probably want to throw in `req.raise_for_status()` here, so if it returns a 404 or whatever, you know immediately.

Reply:
Also nit: prefer `resp` to `req`; what you get back is a Response object, not a Request.